docs/getting-started-guides/docker-multinode/master.sh

```diff
@@ -41,10 +42,10 @@ fi
 
 # Make sure master ip is properly set
 if [ -z ${MASTER_IP} ]; then
-    echo "Please export MASTER_IP in your env"
-    exit 1
+    MASTER_IP=$(hostname -I | awk '{print $1}')
```

Comment:
Why do you default to localhost being the master? In multinode setup that sounds like a strange assumption.